samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -323,12 +342,54 @@ public async Task<string> ExtractAudienceAsync(SKContext context)
 
         await this.UpdateResponseStatusOnClient(chatId, "Calculating total token usage");
         this.GetTokenUsages(context, chatContext);
+
         context.Variables.Update(response);
+
         return context;
     }
 
     #region Private
 
+    /// <summary>
+    /// Invokes AzureContentModerator to analyze image content.
+    /// </summary>
+    /// <returns>SKContext containing the image analysis result.</returns>
+    private async Task<SKContext> ProcessImageContent(
+        [Description("The new message")] string message,
+        [Description("Unique and persistent identifier for the user")] string userId,
+        [Description("Name of the user")] string userName,
+        [Description("Unique and persistent identifier for the chat")] string chatId,
+        SKContext chatContext)
+    {
+        var imageCannedResponse = "Great image!";
```

Comment:
Why do we need to tell the user the image is great?

I can see a bunch of scenarios where saying "Great image!" would be inappropriate.

Reply:
This is no longer relevant - we're treating image as docs, so removed support for image as message + all logic in ChatSkill